apps/emqx_conf/src/emqx_conf_schema_types.erl

```diff
@@ -265,6 +265,12 @@ readable("port_number()") ->
     Result = try_range("1..65535"),
     true = is_map(Result),
     Result;
+readable("secret()") ->
+    #{
+        swagger => #{type => string, example => <<"R4ND0M/S∃CЯ∃T"/utf8>>},
+        dashboard => #{type => string},
+        docgen => #{type => "String", example => <<"R4ND0M/S∃CЯ∃T"/utf8>>}
+    };
```

Comment:
LGTM, but prefer this style: https://github.com/emqx/emqx/pull/11940/commits/3ab858297daa6c67c6ed416e0dcbaf8ebfee40e8